base/applications/iexplore/iexplore.inf

```diff
@@ -28,11 +28,11 @@ HKCR,"CLSID\%CLSID_InternetExplorerManager%\LocalServer32",,,"""%16422%\Internet
 [Settings.Reg]
 HKCU,"Software\Microsoft\Internet Explorer\Main","ShowGoButton",2,"yes"
 #HKCU,"Software\Microsoft\Internet Explorer\Main","Start Page",2,"https://www.winehq.org"
-HKCU,"Software\Microsoft\Internet Explorer\Main","Start Page",2,"http://www.reactos.org"
+HKCU,"Software\Microsoft\Internet Explorer\Main","Start Page",2,"https://reactos.org/"
 HKCU,"Software\Microsoft\Internet Explorer\Main","Search Page",2,"http://www.google.com"
```

Comment:
should we consider using HTTPS for Google as search page too ?

Reply:
That is off-topic in this PR. Make your own PR, if need be...

Reply:
Not needed. Disregard.

Reply:
Filed as #2646.